docs/installation.rst

```diff
@@ -9,7 +9,7 @@ Minimum Requirements
 - Shotgun server v5.4.14 or higher for API v3.0.16+.
 - Shotgun server v2.4.12 or higher for API v3.0.8.
 - Shotgun server v1.8 or higher for API v3.0.7.
-- Python v2.4 - v2.7. (We do not currently support Python 3)
+- Python 2.6 - 2.7 or Python 3.7
```

Comment:
No love for 3.6 or even 3.5? @robblau 
I know VFX reference platform says 3.7, but we've been pretty good so far at casting a wider net. Thoughts?